geoextent/__main__.py

```diff
@@ -28,8 +28,13 @@
 Supported formats:
 - GeoJSON (.geojson)
 - Tabular data (.csv)
-- Shapefile (.shp)
 - GeoTIFF (.geotiff, .tif)
+- Shapefile (.shp)
+- Geopackage (.pkga)
+- GPS Exchange Format (.gpx)
```

Comment:
Text cases for GPX are missing. Please don't forget to add (and skip) tests for time extent extraction.